reposerver/repository/repository.go

```diff
@@ -87,7 +86,7 @@ type Service struct {
 	chartPaths                io.TempPaths
 	gitRepoInitializer        func(rootPath string) goio.Closer
 	repoLock                  *repositoryLock
-	cache                     *reposervercache.Cache
+	cache                     *cache.Cache
```

Comment:
@Fish-pro just my opinion, if we had to get rid of one of them i think I would prefer to keep `reposervercache` because to me the extra bit of verbosity makes it easier to follow the code so I know what part of the code base this cache is related to, maybe just because "cache" is used so frequently that it can get confusing to me to just see that without additional context